content/governance.md

```diff
@@ -46,6 +46,9 @@ Upon death of a member, they leave the team automatically.
 
 In case a member leaves, the [offboarding](#offboarding) procedure is applied.
 
+In case of certified mental incapacitation of a member, the [offboarding](#offboarding) procedure can be applied partially or in full to ensure account and access security and voting rights and quorum count will be suspended.
```

Comment:
Without a vote?

Reply:
Yes. I like to automate cases which emotional load; and it would be emotional for us.

NB: We can always vote to override this case-by-case.

Reply:
This varies a lot based on jurisdiction, and do you mean permanently or just until they're no longer sectioned? This seems very easy to abuse, if it comes to it we can call a vote.

Reply:
I think this is way to blurry to have it in here as anything we can do without a vote.

And if we have to vote on it anyway, we don't really need a separate section.

WRT account and access security: If an account is compromised, it has to be protected. But we don't need to spell that out in the governance.

Reply:
Removed that section.